docs/migration/v8-to-v9.md

```diff
@@ -135,66 +146,78 @@ Older Typescript versions _may_ still work, but we will not test them anymore an
 
   To customize which files are deleted after upload, define the `filesToDeleteAfterUpload` array with globs.
 
 ### `@sentry/react`
 
-The `componentStack` field in the `ErrorBoundary` component is now typed as `string` instead of `string | null | undefined` for the `onError` and `onReset` lifecycle methods. This more closely matches the actual behavior of React, which always returns a `string` whenever a component stack is available.
+- The `componentStack` field in the `ErrorBoundary` component is now typed as `string` instead of `string | null | undefined` for the `onError` and `onReset` lifecycle methods. This more closely matches the actual behavior of React, which always returns a `string` whenever a component stack is available.
 
-In the `onUnmount` lifecycle method, the `componentStack` field is now typed as `string | null`. The `componentStack` is `null` when no error has been thrown at time of unmount.
+  In the `onUnmount` lifecycle method, the `componentStack` field is now typed as `string | null`. The `componentStack` is `null` when no error has been thrown at time of unmount.
 
 ## 3. Package Removals
 
-As part of an architectural cleanup, we deprecated the following packages:
-
-- `@sentry/utils`
-- `@sentry/types`
-
-All of these packages exports and APIs have been moved into the `@sentry/core` package.
-
 The `@sentry/utils` package will no longer be published.
 
-The `@sentry/types` package will continue to be published but it is deprecated and we don't plan on extending its API.
-You may experience slight compatibility issues in the future by using it.
-We decided to keep this package around to temporarily lessen the upgrade burden.
-It will be removed in a future major version.
+The `@sentry/types` package will continue to be published, however, it is deprecated and its API will not be extended.
+It will not be published as part of future major versions.
+
+All exports and APIs of `@sentry/utils` and `@sentry/types` (except for the ones that are explicitly called out in this migration guide to be removed) have been moved into the `@sentry/core` package.
 
-## 4. Removal of Deprecated APIs
+## 4. Removed APIs
 
 ### `@sentry/core` / All SDKs
 
 - **The metrics API has been removed from the SDK.**
 
   The Sentry metrics beta has ended and the metrics API has been removed from the SDK. Learn more in the Sentry [help center docs](https://sentry.zendesk.com/hc/en-us/articles/26369339769883-Metrics-Beta-Ended-on-October-7th).
 
-- The `debugIntegration` has been removed. To log outgoing events, use [Hook Options](https://docs.sentry.io/platforms/javascript/configuration/options/#hooks) (`beforeSend`, `beforeSendTransaction`, ...).
+- The `enableTracing` option was removed.
+  In v9, to emulate `enableTracing: true`, set `tracesSampleRate: 1`.
+  To emulate `enableTracing: false`, remove the `tracesSampleRate` and `tracesSampler` options (if configured).
```

Comment:
```suggestion
  Instead, set `tracesSampleRate: 1` or `tracesSampleRate: 0`.
```

This is good enough IMHO, as even before `tracesSampler` took precedence over `enableTracing`, so that does not change!